docs/source/concept_models/vivarium_ciff_sam_2/concept_model.rst

```diff
@@ -107,17 +113,16 @@ Simulated scenarios will involve some change of coverage/efficacy parameter valu
 
   3. SQ-LNS
 
-    * 3a. Targeted to those with mild wasting
-    * 3b. Targeted to those recovered from SAM treatment
-    * 3c. Targeted to those recovered from MAM treatment
-    * 3d. Universal
+    * 3a. Universal
+    * 3b. Targeted to those with mild wasting
+    * 3c. Targeted to those recovered from SAM/MAM treatment
```

Comment:
```suggestion
    * 3c. Targeted to those recovered from SAM/MAM treatment
    * 3d. No SQ-LNS scale-up
```

Reply:
Or is my 3d your 2?

Reply:
Hmm, I'm thinking that these numbers/letters just describe what the intervention is and then the table below describes the coverage